### DISCUSSION TITLE
Error occurred during compilation using Docker

### DISCUSSION THREAD

**Discussion by zy19970** [alshedivat/al-folio#2508] [Q&A] [answered]
![image](https://github.com/alshedivat/al-folio/assets/22004812/0a727afb-2e6b-4b26-9378-a026c7d07735)

I suspect that my network cannot connect to medium.com. Can I cancel or skip this step?

**Comment by george-gca** [comment, marked as answer]
Maybe this is an error that happens after this step, but I am not sure since the rest of the stacktrace isn't there. If the problem is the external posts part, you can remove the external blog posts entirely. Check [this answer](https://github.com/alshedivat/al-folio/discussions/2279#discussioncomment-8804226).

**Reply by zy19970** [reply]
Thank you. I think Docker is the best way to build it, unless this problem is solved.
